lib/phoenix/router/helpers.ex

```diff
@@ -9,20 +9,26 @@ defmodule Phoenix.Router.Helpers do
   @doc """
   Callback invoked by url generated in each helper module.
   """
-  def url(_router, %Conn{private: private}) do
```

Comment:
Since this is a callback... maybe it should be `__url__`? It seems there is no reason to call it in public, is there?

Reply:
Nevermind. This whole module is `@moduledoc false`. :)